data/systemd/system/yggdrasil-bus@.service

```diff
@@ -5,6 +5,7 @@ Requires=yggdrasil-bus@%i.socket
 
 [Service]
 Type=notify
+User=@user@
```

Comment:
There is no need to explicitly set the `Group=` directive. According to [the documentation](https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#User=):

> If no group is set, the default group of the user is used.

Since the sysuser template creates a user and corresponding "user group" automatically, this group is automatically used by the systemd service. I confirmed this in the output of `ps`:

```
[link@lima-fedora rhc]$ ps axo pid,user,group,args | grep yggd
 472015 link     link     grep --color=auto yggd
3648112 yggdras+ yggdras+ /usr/bin/yggd
3648808 link     link     /usr/bin/yggd
```